src/ForwardShare/ForwardShare.jl

```diff
@@ -136,42 +136,42 @@ module ForwardShare
 		end
 		return nd
 	end
-
-	export getSensTMatVec,getSensMatVec, getSensMatSize, getNumberOfData
-
-	# # ===== Methods for parallelization =====
-	include("getDataParallel.jl")
-	include("prepareMesh2Mesh.jl")
-	include("interpLocalToGlobal.jl")
-	include("getSensMat.jl")
+   
+   
+   export getSensTMatVec,getSensMatVec, getSensMatSize, getNumberOfData
+
+   # # ===== Methods for parallelization =====
+   include("getDataParallel.jl")
+   include("prepareMesh2Mesh.jl")
+   include("interpLocalToGlobal.jl")
+   include("getSensMat.jl")
 	include("testing.jl")
 
-	import jInv.Utils.clear!
-	function clear!(P::ForwardProbType;clearAinv::Bool=true,clearFields::Bool=true, clearMesh::Bool=false, clearSources::Bool=false, clearObs::Bool=false,clearAll::Bool=false)
-		if clearAll || clearMesh
+   import jInv.Utils.clear!
+   function clear!(P::ForwardProbType;clearAinv::Bool=true,clearFields::Bool=true, clearMesh::Bool=false, clearSources::Bool=false, clearObs::Bool=false,clearAll::Bool=false)
+      if clearAll || clearMesh
 			P.M = clear!(P.M)
-		end
-		if clearAll || clearSources
-			P.Sources = clear!(P.Sources)
-		end
-		if clearAll || clearObs
-			P.Obs     = clear!(P.Obs)
-		end
-		if clearAll || clearFields
-		if any(fieldnames(P) .== :Fields) 
-			P.Fields = clear!(P.Fields)
-		end
-
-	end
-
-	if clearAll || clearAinv
-	if any(fieldnames(P) .== :Ainv) 
-	   clear!(P.Ainv)
-	end
-	end
-
-	
-end
+      end
+      if clearAll || clearSources
+         P.Sources = clear!(P.Sources)
+      end
+      if clearAll || clearObs
+         P.Obs     = clear!(P.Obs)
+      end
+      if clearAll || clearFields
+         if any(fieldnames(P) .== :Fields) 
+            P.Fields = clear!(P.Fields)
+         end
+         
+      end
+      
+      if clearAll || clearAinv
+         if any(fieldnames(P) .== :Ainv) 
+            clear!(P.Ainv)
+         end
+      end
+      
+   end  # function clear!
 
 
-end
+end  # module ForwardShare
```

Comment:
I don't anything changed in ForwardShare.jl?  If I'm right about that can we remove it from the pr?